.github/workflows/engine_test.yml

```diff
@@ -43,13 +43,53 @@ jobs:
         python-version:  ${{ matrix.python-version }}
     - name: Install dependencies
       run: |
-        python install.py devel
-    - name: hazardlib tests
+        OQ_APPLICATION_MODE=public python install.py devel
```

Comment:
install.py ignores OQ_APPLICATION_MODE so I wonder why are you setting the variable here

Reply:
Here it was a leftover. I am trying to remove it also from the server tests and see how it behaves. In that case I expect things to fail, because the application mode should affect how the Django application is installed (running the necessary migrations if users are required, for instance).

Reply:
You're right, I was making a little confusion about it.